jekyll-assets/_layouts/boxes404.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    {% include head.html %}
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/boxes.css">
+  </head>
+  <body>
+    {% include fonts.html %}
+    {% include header.html %}
+    {% include toptitle404.html %}
+    {% include subtitle404.html %}
+    {% include tabs.html %}
+
+    <div id="container">
+      <section id="box-content">
+        {% for item in site.data.index.tabs %}
+          {% if page.dir == item.path or page.dir == "/" and item.default_tab and item.default_tab == "yes" %}
+            {% for entry in item.subitems %}
+            {% if entry.path %}
+            <a class="box" href="{{ site.baseurl }}{{ entry.path }}">
+            {% else %}
+            <a class="box" href="{{ entry.url }}">
+            {% endif %}
+              <span><img src="{{ site.baseurl }}{{ entry.imagepath }}"></span>
+              <span class="title">{{ entry.title }}</span>
+              <span>{{ entry.description }}</span>
+            </a>
+            {% endfor %}
+          {% endif %}
+        {% endfor %}
+      </section>
+    </div>
```

Comment:
Given this section now appears in two layouts, is it worth extracting this into `_includes` and then using it both here and in the `boxes.html` layout? That might be a nice balance between reducing duplication and keeping this explicit?

Reply:
I'll leave these restructuring-decisions to @nelliemckesson as she came up with the original design, and is much more familiar with Jekyll than me!

Reply:
Have a look at the latest commit - I've solved the "duplication problem" in a simpler way.